spatial_maps/stats.py

```diff
@@ -158,7 +158,7 @@ def prob_dist(x, y, bins):
     '''
 
     H, _, _ = np.histogram2d(x, y, bins=bins, normed=False)
-    return (H / len(x)).T
+    return (H / len(x))
```

Comment:
Are you sure? I think this was correct before (with transpose)

Reply:
it was correct before because the x, y was inverted when creating the maps. Now you don't have to transpose anymore. Quite sure about this as the rate_map and prob_dist have the same size (non-square in my case)

Reply:
:+1: